apps/src/sites/studio/pages/scriptOverview.js

```diff
@@ -18,7 +19,10 @@ function initPage() {
   const store = getStore();
 
   if (plcBreadcrumb) {
-    renderPlcBreadcrumb(plcBreadcrumb, document.getElementById('breadcrumb'));
+    // Dispatch breadcrumb props so that ScriptOverviewHeader can add the breadcrumb
+    // as appropriate
+    registerReducers({plcHeader: plcHeaderReducer});
```

Comment:
It might be a little weird to conditionally register this reducer, but it seemed better than having a reducer that isn't used on 99% of pages.